### PR TITLE
soccer_interfaces: 0.0.1-1 in 'galactic/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by ijnek**
Increasing version of package(s) in repository `soccer_interfaces` to `0.0.1-1`:

- upstream repository: https://github.com/ijnek/soccer_interfaces.git
- release repository: https://github.com/ijnek/soccer_interfaces-release.git
- distro file: `galactic/distribution.yaml`
- bloom version: `0.10.7`
- previous version for package: `null`

### CODE REVIEW

**Comment by ijnek** [comment]
Hi @cottsay, did you happen to get a chance to look at this PR? Thanks.

**Comment by ahcorde** [comment]
CI is failing.

Can you add `source` and `doc` fields ?

**Review by ahcorde — DISMISSED** [review]
CI is failing.

Can you add `source` and `doc` fields ?

**Comment by cottsay** [comment]
After taking a look, I think the CI problem is actually on our side, and is related to the rosdep key checking automation:
```
fatal: remotes/origin/master...HEAD: no merge base
```

The problem seems to be related to shallow cloning, though I'm confused as to why it's happening here and not on other PRs.

I merged master back into the source branch, which should work around the problem for now.

File: galactic/distribution.yaml [hunk 1]
```diff
@@ -2988,6 +2988,15 @@ repositories:
       url: https://github.com/SteveMacenski/slam_toolbox.git
       version: galactic
     status: maintained
+  soccer_interfaces:
+    release:
```
Comment (jacobperron) [inline galactic/distribution.yaml:2992]: Consider adding a `source` entry to make it easier for people to find the source repository.

Comment (ijnek) [reply galactic/distribution.yaml:2996]: Done.

**Comment by ijnek** [comment]
@ahcorde 
Thanks for the review, I've added ``source`` and ``doc`` fields as per your suggestion

File: galactic/distribution.yaml [hunk 2]
```diff
@@ -3160,6 +3160,23 @@ repositories:
...
+      - soccer_vision_msgs
+      tags:
+        release: release/galactic/{package}/{version}
+      url: https://github.com/ijnek/soccer_interfaces-release.git
+      version: 0.0.1-1
+    source:
+      type: git
+      url: https://github.com/ijnek/soccer_interfaces.git
+      version: main
+    status: developed
```
Comment (jacobperron) [inline galactic/distribution.yaml:3179]:
```suggestion
      version: main
    status: developed
```

**Pull request merged**